### PR TITLE
rename round name from round_name to current_round_name in code_trek_…

### PR DESCRIPTION
**Pull request by thisajaychauhan**
Targets #3076 

### Description

Rename round_name column from **round_name** to **current_round_name** in **code_trek_applicant** table.

### Checklist:
- [x] I have performed a self-review of my own code.

### CODE REVIEW

File: database/migrations/2023_05_17_114304_rename_round_name_column_of_code_trek_applicants_table.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('code_trek_applicants', function (Blueprint $table) {
+            $table->renameColumn('round_name', 'current_round_name');
```
Comment (GauravGusain98) [inline database/migrations/2023_05_17_114304_rename_round_name_column_of_code_trek_applicants_table.php:17]:
```suggestion
            $table->renameColumn('round_name', 'latest_round_name');
```

File: database/migrations/2023_05_17_114304_rename_round_name_column_of_code_trek_applicants_table.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('code_trek_applicants', function (Blueprint $table) {
+            $table->renameColumn('current_round_name', 'round_name');
```
Comment (GauravGusain98) [inline database/migrations/2023_05_17_114304_rename_round_name_column_of_code_trek_applicants_table.php:29]:
```suggestion
            $table->renameColumn('latest_round_name', 'round_name');
```

File: database/migrations/2023_05_17_114304_rename_round_name_column_of_code_trek_applicants_table.php (hunk 1 above)
Comment (GauravGusain98) [reply database/migrations/2023_05_17_114304_rename_round_name_column_of_code_trek_applicants_table.php:17]: You also need to correct the logics where we were using this column

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
@thisajaychauhan  As far as I can see, the `round_name` column is used everywhere in the CodeTrek Modules. In the CodeTrek module, we are fetching and storing the data by round_name. There is a need to change there too.

File: database/migrations/2023_05_18_132943_rename_current_round_name_column_of_code_trek_applicants_table.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('code_trek_applicants', function (Blueprint $table) {
+            $table->renameColumn('latest_round_name', 'current_round_name');
+        });
```
Comment (Ayush8923) [inline database/migrations/2023_05_18_132943_rename_current_round_name_column_of_code_trek_applicants_table.php:30]: Where did you create this second migration file? It is not needed. You can make changes in the migration `rename_round_name_column_of_code_trek_applicants_table` file that you created earlier.

File: Modules/HR/Database/Seeders/SettingsTableSeeder.php [hunk 4]
```diff
@@ -67,7 +67,7 @@ public function run()
...
-            'setting_value' => '<div>Hello |*applicant_name*|,</div><br></div><div>Hope you are doing well and having a great time. We are reaching out to follow up on the interview scheduling process. You must have received an email for scheduling your interview for |*round_name*|. The email would have a link, that would take you to a calendar page, with available slots for booking. Pick one as per your convenience and schedule your interview.<br>If you didn\'t find an email, please check your spam folder. For further queries, write at pankaj.kandpal@coloredcow.in<br>
+            'setting_value' => '<div>Hello |*applicant_name*|,</div><br></div><div>Hope you are doing well and having a great time. We are reaching out to follow up on the interview scheduling process. You must have received an email for scheduling your interview for |*latest_round_name*|. The email would have a link, that would take you to a calendar page, with available slots for booking. Pick one as per your convenience and schedule your interview.<br>If you didn\'t find an email, please check your spam folder. For further queries, write at pankaj.kandpal@coloredcow.in<br>
```
Comment (Ayush8923) [inline Modules/HR/Database/Seeders/SettingsTableSeeder.php:70]: THis file is not needed. Please revert this.

File: Modules/HR/Config/config.php [hunk 5]
```diff
@@ -96,7 +96,7 @@
         'applicant-name' => '|*applicant_name*|',
         'interview-time' => '|*interview_time*|',
         'job-title' => '|*job_title*|',
-        'round-name' => '|*round_name*|'
+        'round-name' => '|*latest_round_name*|'
```
Comment (Ayush8923) [inline Modules/HR/Config/config.php:99]: Same here.

File: Modules/CodeTrek/Services/CodeTrekService.php [hunk 6]
```diff
@@ -76,7 +76,7 @@ public function moveApplicantToRound($applicant, $data)
     {
         $applicationRound = new CodeTrekApplicantRoundDetail();
         $applicationRound->applicant_id = $applicant->id;
-        $applicationRound->round_name = 'level-1';
+        $applicationRound->current_latest_round_name = 'level-1';
```
Comment (Ayush8923) [inline Modules/CodeTrek/Services/CodeTrekService.php:79]:
```suggestion
        $applicationRound->latest_round_name = 'level-1';
```

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments. Please check.
And please test locally before assigning the for the review.

File: Modules/HR/Config/config.php (hunk 5 above)
Comment (Ayush8923) [inline Modules/HR/Config/config.php:99]: Can you please revert this file? This file is not needed.

File: Modules/HR/Database/Seeders/SettingsTableSeeder.php (hunk 4 above)
Comment (Ayush8923) [inline Modules/HR/Database/Seeders/SettingsTableSeeder.php:70]: Same here. Please revert this file. It's not needed.

**Review by Ayush8923 — CHANGES_REQUESTED** [review]
Left some comments. 
Please check.

**Review by Ayush8923 — COMMENTED** [review]
Have you tested all the areas in CodeTrek Module?

**Review by rahulpanwar16 — APPROVED** [review]
Looks good to me.

**Review by Ayush8923 — APPROVED** [review]

**Review by GauravGusain98 — APPROVED** [review]
lgtm

**Pull request merged**